docs/reference/indices/flush.asciidoc

```diff
@@ -39,7 +40,153 @@ or even on `_all` the indices.
 
 [source,js]
 --------------------------------------------------
-$ curl -XPOST 'http://localhost:9200/kimchy,elasticsearch/_flush'
+POST /kimchy,elasticsearch/_flush
 
-$ curl -XPOST 'http://localhost:9200/_flush'
+POST /_flush
 --------------------------------------------------
+// AUTOSENSE
+
+[[indices-synced-flush]]
+=== Synced Flush
+
+Elasticsearch tracks the indexing activity of each shards. Shards that have not
+received any indexing operations for 30 minutes (configurable) are automatically marked as inactive. This presents
+an opportunity for Elasticsearch to reduce shard resources and also perform
+a special kind of flush, called `synced flush`. A synced flush performs normal
+flushing and adds a special uniquely generated marker (`sync_id`) to all shards.
```

Comment:
A synced flush performs a normal flush, then adds a generated unique marker (`sync_id`) to all shards.